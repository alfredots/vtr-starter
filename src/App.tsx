function App() {
  return (
    <div className="h-screen w-screen flex items-center justify-center">
      <h1 className="text-xl lg:text-3xl font-bold">
        Vite + <span className="text-purple-600">Tailwind</span>
      </h1>
    </div>
  )
}

export default App
