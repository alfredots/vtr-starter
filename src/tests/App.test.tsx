import { render, screen } from '@testing-library/react'
import { expect, it } from 'vitest'
import App from '../App'

it('should render a App', () => {
  const { container } = render(<App />)
  expect(screen.getByText(/Vite +/i)).toBeDefined()
  expect(screen.getByText(/Tailwind/i)).toBeDefined()
  expect(container.firstChild).toMatchSnapshot()
})
