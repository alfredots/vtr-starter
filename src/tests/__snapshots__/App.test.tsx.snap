// Vitest Snapshot v1

exports[`should render a App 1`] = `
<div
  class="h-screen w-screen flex items-center justify-center"
>
  <h1
    class="text-xl lg:text-3xl font-bold"
  >
    Vite + 
    <span
      class="text-purple-600"
    >
      Tailwind
    </span>
  </h1>
</div>
`;
